de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/Wikipedia.java

```diff
@@ -51,6 +51,8 @@
 // TODO better JavaDocs!
 public class Wikipedia implements WikiConstants {
 
+    static final String SQL_COLLATION = "COLLATE utf8_bin";
```

Comment:
I'd suggest to include the space at the beginning of this string instead of at the end of the query (where it may more easily be forgotten).

Reply:
@reckart will fix it as suggested and push ASAP.